src/pages/gallery/components/CollectionDropZone.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import styled from 'styled-components';
+import UploadService from 'services/uploadService';
+import { fetchData } from 'services/fileService';
+
+const getColor = (props) => {
+    if (props.isDragAccept) {
+        return '#00e676';
+    }
+    if (props.isDragReject) {
+        return '#ff1744';
+    }
+    if (props.isDragActive) {
+        return '#2196f3';
+    }
+};
+
+const enableBorder = (props) => (props.isDragActive ? 'dashed' : 'none');
+
+const DropDiv = styled.div`
+  width:33%;
+  border-width: 2px;
+  border-radius: 2px;
+  border-color: ${(props) => getColor(props)};
+  border-style: ${(props) => enableBorder(props)};
+  outline: none;
+  transition: border 0.24s ease-in-out;
+`;
+
+const CollectionDropZone = ({
```

Comment:
[question]
Why do we need two dropzone components(FullScreenDropzone and CollectionDropZone)?

[change & note for future]
Also, avoid using the fat arrow function for creating a component. Instead, use functions.
Fat arrow functions are bound to the lexical scope and we don't use this inside a component so an extra function is getting created unnecessarily.

Reply:
The FullScreenDropzone is just for trigerring the onDrag event on the whole screen It does nothing else. The collectionDropzone handles all the operation. It could be replaced by just a normal elemnt with onDragOver will do that.....didnt know about the HTML onDragOver event earlier thought it was something that React-dropzone provided.